requirement/production.txt

```diff
@@ -9,3 +9,4 @@ gunicorn==19.7.1
 whitenoise==3.3.1
 django-markdown-deux==1.0.5
 psycopg2==2.7.5
+requests==2.19.1
```

Comment:
FYI, I have requests==2.18.4 because higher versions clash with a requirement from localstack (which is defined as a development dependency to test locally the static asset collection and serving from S3). Any strong reason for using 2.19?

Reply:
`2.19.1` is the latest version, that's all. This branch needs to be reconciled with what you have now in master.

---

I took a look at master, and `django-storages` certainly fits the situation better than `whitenoise`.

Running a dummy S3 in development is cool, or anyway preferable to maintaining a separate configuration of static asset management. Though, bear in mind, the filesystem-based configuration is quite easy, and can skip over `collectstatic` and such entirely.

Reply:
`whitenoise` and `localstack` are cool solutions for certain needs.

But is the only problem we're trying to solve the serving of static assets during automated tests? Or, do you mean with manual tests, with debug mode off? In either case, you definitely don't *need* either solution; but, I might be missing precisely what you want to achieve. (I've certainly developed Django apps which use the filesystem in dev and S3 in prod.)